README.md

```diff
@@ -19,6 +19,59 @@ First-time users should go through the [getting-started](https://docs.vespa.ai/e
 Explore the [examples](examples) for smaller applications helping you getting started with a particular feature,
 and see [operations](examples/operations) for operational examples.
 
+## Standard structure of a Sample Application
+
+We try to build each sample application with a standard structure, so you can
+ easily find your way around. This section gives you an overview of this structure.
+
+<details>
+<summary>Click to expand</summary>
+
+The standard structure is as follows:
+
+```plaintext
+one-sample-app/
+├─ README.md
+├─ dataset/
+│  ├─ a-document.json
+│  ├─ another-doc.json
+│  ├─ ...
+├─ app/
+│  ├─ security/
+│  ├─ schemas/
+│  │  ├─ my_vespa_schema.sd
+│  ├─ services.xml
+│  ├─ ...
+├─ script/
+│  ├─ build.sh             # when required
+│  ├─ ...
+```
+
+There might be other files and folders than the ones listed here, but this should be the bare minimum that you can expect.
+
+Let's analyze the structure:
+
+- `README.md` - self explainatory: the README specific for the sample application, we recommend reading it through.
+- `dataset/` - contains the dataset used for the sample application. This is usually a small set of documents, but it can be larger.
+
+  **Note:** Sometimes this might be missing, and will be generated by the build script (see below), this is usually because the dataset is too large to be included in the repository.
+-  `app/` - contains the [Vespa application package](https://docs.vespa.ai/en/application-packages.html), this is the actual definition of the Vespa deployment. The bare minimum includes:
+    - `security/` - this is where you will place your `clients.pem` certificate for data-plane authentication. _(Refer to our [Security Guide](https://cloud.vespa.ai/en/security/guide.html#application-configuration))_
+    - `schemas/` - contains [the schema](https://docs.vespa.ai/en/schemas.html) files used for the application. This is usually a single file, but it can be multiple files.
+    - `services.xml` - contains the services configuration for the application. This is usually a single file.
+    - Other files and folders that are required for the application to run.
+- `script/` - contains the optional scripts used to help building the application package. This is usually a single file, but it can be multiple files.
+  - `build.sh` - this is the main script, if multiple files are present, this will be the only one you'll need to worry about.
+
+    **Note:** We recomend refering to the README for the sample application for details on how to run it.
+
+
+> [!NOTE]
```

Comment:
What does `[!xx]` syntax mean?

Reply:
I am pretty sure this is special for Github: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
Neat. I ask because I tried to open it from the commit tree, but it didn't render at all https://github.com/vespa-engine/sample-apps/blob/efad8d1dcc1179182c01d654d1a34b388cdeb4a7/README.md

Reply:
After further investigation it seems that GH does NOT support this in markdown documents, but only in their own makrdown fields (issues, prs, ...)